src/env.ts

```
@@ -1,58 +1,58 @@
-   * This means that if you have an empty string for a value that is supposed
-   * to be a number (e.g. `PORT=` in a ".env" file), Zod will incorrectly flag
-   * it as a type mismatch violation. Additionally, if you have an empty string
-   * for a value that is supposed to be a string with a default value (e.g.
-   * `DOMAIN=` in an ".env" file), the default value will never be applied.
-   *
-   * In order to solve these issues, we recommend that all new projects
-   * explicitly specify this option as true.
-   */
-  emptyStringAsUndefined: true,
-});
+import { createEnv } from '@t3-oss/env-core';
```

ESLint

[error] Unable to resolve path to module '@t3-oss/env-core'.

(import-x/no-unresolved)